src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -199,7 +199,8 @@ ss::future<response_ptr> create_topics_handler::handle(
     }
 
     if (!ctx.audit()) {
-        request.data.topics.erase(valid_range_end, request.data.topics.end());
```

Comment:
chunked_vector does not have `erase()`

Reply:
I wonder if an `chunked_vector::erase_to_end(iterator)` would be an easier drop in replacement instead of `pop_back_n`.

Reply:
Yes, I had the same thought and this comment pushed me over the the edge to add it, it's in this PR as of the latest push.